### DISCUSSION TITLE
Пробел только для воспроизведения/паузы?

### DISCUSSION THREAD

**Discussion by forjosis** [cawa-93/anime-library#127] [Новые идеи]
Добрый день!)

Сейчас если нажать мышкой на иконку «Во весь экран», «Отключить звук», а потом нажать на пробел, то пробел будет управлять последним нажатым действием. Возможно было бы удобно оставить пробел только для воспроизведения и паузы.

Откуда такая идея: если развернул экран мышкой, потом понадобилось срочно нажать на паузу (иногда некоторые команды делают слишком громкие вставки.), то вместо паузы приложение выходит из полноэкранного режима, но продолжает воспроизводить видео.

Я знаю, что можно использовать клавишу «f» вместо кнопки. Но когда сначала делаешь всё мышкой (открываешь приложение, выбираешь аниме, озвучку), то для входа в полноэкранный режим тоже используешь мышь.

У Ютуб также, там это тоже может мешать. У плеера в смотреть аниме, кстати, пробел не повторяет действия кнопок.

Возможно для этого приложения это было бы тоже удобно)

**Comment by forjosis** [comment]
Да, что можно использовать «k» тоже знаю) Пробел просто интуитивней, заметней, удобнее и привычней )

**Reply by cawa-93** [reply]
Это происходит от специфики платформы. После нажатия по кнопке эта кнопка остаётся "В фокусе" и пробел уплавляет прежде всего кнопкой "в фокусе". Это сделано для того, чтобы была возможность управлять всем плеером с клавиатуры: Клавишей `Tab` вы можете переключать фокус а пробелом и стрелками управлять выбранным контролом. 

Я понимаю, что это может быть немного не удобно (после клика по кнопке нужно кликнуть куда-то ещё чтобы снять фокус на кнопке) но это вынужденная мера и лучшего подхода я пока не вижу.

**Reply by forjosis** [reply]
Спасибо за быстрый ответ)
Ага, понимаю ситуацию с фокусом. Это мелочь. Если в будущем можно будет найти подход, то мне кажется, будет отлично. В любом случае, приложение от этого не страдает)
